README.md

```diff
@@ -150,28 +137,35 @@ To run the pipeline, enter the following command into the terminal:
 |
 |   create_publication.py             <- Runs the overall pipeline to produce the publication     
 |
++---data_in                           <- Stores imported data that will be used to create the outputs
+|   |       .gitkeep                  <- Empty file that enables the data_in folder to be committed
++---data_out                          <- Outputs and aggregations created by the pipeline are stored here
+|   |       .gitkeep                  <- Empty file that enables the data_in folder to be committed
 +---src                               <- Scripts with functions for use in 'create_publication.py'. Contains project's codebase.
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |
-|   +---utils                     <- Scripts relating to configuration and handling data connections e.g. importing data, writing to a database etc.
+|   +---utils                         <- Scripts relating to configuration and handling data connections e.g. importing data, writing to a database etc.
 |   |       __init__.py               <- Makes the functions folder an importable Python module
 |   |       file_paths.py             <- Configures file paths for the package
 |   |       logging_config.py         <- Configures logging
-|   |       data_connections.py       <- Handles data connections i.e. reading/writing dataframes from SQL Server
+|   |       spark.py                  <- For setting up and configuring Apache Spark
```

Comment:
I wonder if we should have put this within the data_connections file...

Reply:
The spark.py file currently only has creating the spark session at present, not really a data connection exactly. In the data_ingestion folder there is reading_data.py which creates the dataframe from the csv